src/solver/CMakeLists.txt

```diff
@@ -109,15 +70,16 @@ macro(solver_app name swap)
 		${SRCS}
 		main.h main.hxx main.cpp
 	)
-
-	set(ANTARES_SOLVER_LIBS
+	
+	set(ANTARES_SOLVER_LIBS		
 		libantares-core-calendar
 		libantares-license
-		yuni-static-core
 		yuni-static-uuid
+		yuni-static-core		
 		${CMAKE_THREADS_LIBS_INIT}
 	)
 
+	#TODO : check for windows

```

Comment:
can you add few details about should be checked?

Reply:
TODO removed because this is dead code. 
_wsock32_ is added with `import_std_libs `macro